### PR TITLE
Fixes #22036: Import directive API fail for user technique

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/22036

### CODE REVIEW

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Review by fanf — APPROVED** [review]
Good work!

**Comment by fanf** [comment]
OK, squash merging this PR

File: webapp/sources/rudder/rudder-core/src/main/resources/ldap/bootstrap.ldif [hunk 1]
```diff
@@ -251,6 +251,16 @@ acceptationTimestamp: {"1.0":"20110715124328.999Z"}
...
 
+# test_import_export_archive
+dn: activeTechniqueId=test_import_export_archive,techniqueCategoryId=Rudder Internal,techniqueCategoryId=Active Techniques,ou=Rudder,cn=rudder-configuration
+objectClass: activeTechnique
+objectClass: top
+techniqueId: test_import_export_archive
+activeTechniqueId: test_import_export_archive
+acceptationTimestamp: {"1.0":"20110715124328.999Z"}
+isEnabled: TRUE
+isSystem: FALSE
```
Comment (ncharles) [inline webapp/sources/rudder/rudder-core/src/main/resources/ldap/bootstrap.ldif:262]: this should have been in the tests

**Pull request merged**